XCUITests/SearchTest.swift

```diff
@@ -0,0 +1,167 @@
+    }
+    
+    private func typeOnSearchBar(text: String) {
+        app.textFields["url"].tap()
+        app.textFields["address"].typeText(text)
+    }
+    
+    func testPromptPresence() {
+        // Suggestion is off by default, so the prompt should appear
+        typeOnSearchBar(text: "foobar")
+        waitforExistence(app.staticTexts[LabelPrompt])
+        
+        // No suggestions should be shown
+        waitforNoExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Enable Search suggestion
+        app.buttons["Yes"].tap()
+        
+        // Suggestions should be shown
+        waitforExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Verify that previous choice is remembered
+        app.buttons["Cancel"].tap()
+        typeOnSearchBar(text: "foobar")
+        waitforExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Reset suggestion button, set it to off
+        app.buttons["Cancel"].tap()
+        suggestionsOnOff()
+        typeOnSearchBar(text: "foobar")
+        
+        // Suggestions prompt should not appear
+        waitforNoExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+    }
+    
+    func testDismissPromptPresence() {
+        typeOnSearchBar(text: "foobar")
+        waitforExistence(app.staticTexts[LabelPrompt])
+        
+        app.buttons["No"].tap()
+        waitforNoExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Verify that it is possible to enable suggestions after selecting No
+        app.buttons["Cancel"].tap()
+        suggestionsOnOff()
+        typeOnSearchBar(text: "foobar")
+        waitforExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+    }
+    
+    func testDoNotShowSuggestionsWhenEnteringURL() {
+        // According to bug 1192155 if a string contains /, do not show suggestions, if there a space an a string, the suggestions are shown again
+        typeOnSearchBar(text: "foobar")
+        waitforExistence(app.staticTexts[LabelPrompt])
+        
+        // No suggestions should be shown
+        waitforNoExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Enable Search suggestion
+        app.buttons["Yes"].tap()
+        
+        // Suggestions should be shown
+        waitforExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Typing / should stop showing suggestions
+        app.textFields["address"].typeText("/")
+        waitforNoExistence(app.tables["SiteTable"].buttons[SuggestedSite])
+        
+        // Typing space and char after / should show suggestions again
+        app.textFields["address"].typeText(" b")
+        waitforExistence(app.tables["SiteTable"].buttons["foobar burn cd"])
+    }
+    
+    func testCopyPasteComplete() {
+        // Copy, Paste and Go to url
+        typeOnSearchBar(text: "www.mozilla.org")
```

Comment:
This is similar, I tried just to write in the search bar and copy the text without going to the url, it is not needed. Just checking that I can write, copy and later paste